pyproject.toml

```diff
@@ -69,7 +69,7 @@ PyYAML = "*"
 refgenconf = ">=0.12.0"
 requests = "*"
 Routes = "*"
-social-auth-core = {version = "==3.3.0", extras = ["openidconnect"]}
+social-auth-core = {version = "==4.0.3", extras = ["openidconnect"]}
```

Comment:
@nsoranzo Do we also need to add this to `lib/galaxy/dependencies/pinned-requirements.txt` and `lib/galaxy/dependencies/dev-requirements.txt` ?

Reply:
Yep, I was going to run Poetry locally and push the relevant changes. Or we can merge and wait for https://github.com/galaxyproject/galaxy/pull/12916 to pick it up.

Reply:
Does https://github.com/galaxyproject/galaxy/pull/12916 run against 21.09 as well ?

Reply:
Ops, no, I didn't notice the target branch, sorry.

Reply:
@mvdbeek @nsoranzo Please let me know if I need to update the mentioned files above

Reply:
After checking that the change is compatible with the other requirements in these files, I've pushed the change to this PR.